### PR TITLE
feat: [IOBP-309, IOBP-312] Add error handling for payment verification and activation requests

### PR DESCRIPTION
**Pull request by mastro993**
## Short description
This PR adds the handling of failures during the verification and activation requests for a payment.

## List of changes proposed in this pull request
- Added `WalletPaymentFailure` type
- Added `WalletPaymentFailureDetail` component to display payment failures
- Added `usePaymentFailureSupportModal` hook to display the payment support request bottom sheet modal
- Added required locale keys

## How to test
Using the io-dev-api-server, checkout this branch https://github.com/pagopa/io-dev-api-server/pull/335 and add the following to the config.json:
```json
 "features": {
    "wallet": {
      "verificationFailure": {
        "faultCodeCategory": "PAYMENT_UNAVAILABLE",
        "faultCodeDetail":"PPT_PSP_SCONOSCIUTO"
      }
    }
  }
 ```
 With the IO app, navigate to the wallet payment playground and check that the failure is correctly displayed.

## Preview

https://github.com/pagopa/io-app/assets/6160324/03ec83b8-68c5-4d77-83a1-3bfd7dd8afcd

### CODE REVIEW

**Review by Hantex9 — CHANGES_REQUESTED** [review]
Beyond the following small change, there seems to be strange a behavior in `OperationResultContent` component where the close button is present as shown below:

File: ts/features/walletV3/payment/hooks/usePaymentFailureSupportModal.tsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+  const { bottomSheet, present, dismiss } = useIOBottomSheetAutoresizableModal({
+    component: (
+      <>
+        <ListItemHeader label={I18n.t("wallet.payment.support.supportTitle")} />
+        <ListItemAction
+          label={I18n.t("wallet.payment.support.phone", {
+            phoneNumber: "06.4520.2323"
+          })}
+          accessibilityLabel={I18n.t("wallet.payment.support.phone", {
+            phoneNumber: "06.4520.2323"
```
[the remark this answers is not in the document]
Comment (mastro993) [reply ts/features/walletV3/payment/hooks/usePaymentFailureSupportModal.tsx]: Fixed in https://github.com/pagopa/io-app/pull/5378/commits/bce65ad25057c9b8ceae52955ac5990002243368

**Comment by mastro993** [comment]
> 141c5f7

Fixed in https://github.com/pagopa/io-app/pull/5378/commits/141c5f7959bbe568745f99b2108d868683840b1d

**Comment by mastro993** [comment]
> Beyond the following small change, there seems to be strange a behavior in `OperationResultContent` component where the close button is present as shown below:
> 
> 

Fixed in https://github.com/pagopa/io-app/pull/5378/commits/141c5f7959bbe568745f99b2108d868683840b1d

**Review by Hantex9 — APPROVED** [review]
LGTM!

**Pull request merged**